samples/stc/edit.cpp

```diff
@@ -481,11 +504,50 @@ void Edit::OnCharAdded (wxStyledTextEvent &event) {
         SetLineIndentation (currentLine, lineInd);
         GotoPos(PositionFromLine (currentLine) + lineInd);
     }
+    else if (chr == '#') {
+        wxString s = "define?0 elif?0 else?0 endif?0 error?0 if?0 ifdef?0 "
+                     "ifndef?0 include?0 line?0 pragma?0 undef?0";
+        AutoCompShow(0,s);
+    }
+}
+
+void Edit::OnCallTipClick(wxStyledTextEvent &event)
+{
+    if ( event.GetPosition() == 1 ) {
```

Comment:
```suggestion
    if ( event.GetPosition() == 1 )
    {
```
(also in a few more places below)

Reply:
This was to match the code style in the surrounding methods. Should I change it to match the current code guidelines?

Reply:
I'll have to make a new commit to address the other 3 comments.  Should I change these at that time as well or should I leave them to match the surrounding methods?

Reply:
I haven't realized this was to match the existing style. I don't really know what to do in such cases, perhaps the best would be to commit this as is and then reformat it all at once in a later commit? In any case, please feel free to ignore this suggestion.